crates/uv/src/cli.rs

```diff
@@ -2085,6 +2089,18 @@ pub(crate) struct ToolchainInstallArgs {
     pub(crate) target: Option<String>,
 }
 
+#[derive(Args)]
+#[allow(clippy::struct_excessive_bools)]
+pub(crate) struct ToolchainFindArgs {
+    /// The version to find.
+    #[arg(long)]
+    pub(crate) version: Option<String>,
```

Comment:
I somewhat expected this to just take `--python`. Why this schema?

Reply:
Yeah uhh I was thinking it might be nice to present actual options just for usability and I'd provide a `--python`-like argument too. Maybe we should just take a positional argument that's treated like `--python`?